recipe/meta.yaml

```diff
@@ -20,12 +20,12 @@ requirements:
     - python
     - pip
   run:
-    - python
-    - asteval>=0.9.11
-    - numpy
-    - scipy>=0.17
+    - python>=3.5
+    - asteval>=0.9.16
+    - numpy>=1.16
+    - scipy>=1.2
     - six
-    - uncertainties>=3.0
+    - uncertainties>=3.0.1
```

Comment:
Conda-forge community recommends to use a blank space between the package name and the comparisson with version (only before >, <, =, not after). Could you please fix the recipe?

Reply:
I actually followed the pattern. The version numbers are the minimal versions from Lmfit documentation.

Reply:
The pattern was originally wrong :)